src/main/java/net/homework/webapp/model/Department.java

```diff
@@ -0,0 +1,41 @@
+package net.homework.webapp.model;
+
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.Set;
+
+@Entity
+@Builder
+@Table(name = "department")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class Department {
+
+    @Id
+    @Column(name = "department_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name")
+    private String departmentName;
+
+    private String location;
+
+    @JsonManagedReference
```

Comment:
why do you need this annotation?

Reply:
did not remove because it does not work without it